nightmare_firmware/nightmare_firmware.ino

```diff
@@ -38,7 +39,58 @@ void setup(){
 }
 
 void loop(){
-	
+	/* esp32 bound packet (ROS -> esp32):
```

Comment:
Maybe the packet handling should be done in a function of it's own, @Cogno-Marco what do you think?

Reply:
Might be a good idea yeah, but I don't feel like it's strictly necessary, yet.

Reply:
Ok, if other logic not related to receiving the package is added we will separate this in multiple functions then